paths:
  source_sensor_data_directory: "/home/benedikt/Documents/ArchiveWork/Datasets/a2d2.audi/Ingolstadt"
  source_bus_signals_filepath: "/home/benedikt/Documents/ArchiveWork/Datasets/a2d2.audi/Ingolstadt/bus/20190401145936_bus_signals.json"
  source_sensor_configuration_filepath: "/home/benedikt/Documents/ArchiveWork/Datasets/a2d2.audi/Ingolstadt/cams_lidars.json"

  target_directory: "/home/benedikt/Documents/ArchiveWork/Datasets/a2d2.audi/Ingolstadt_output"

filter:
  #start_timestamp: 1533906480
  stop_timestamp: 1554121568
